build.gradle

```diff
@@ -177,6 +177,7 @@ license {
 	exclude "**/*.json"
 	exclude "**/*.xml"
 	exclude "assets/*"
+	exclude "cubicchunks/server/chunkio/async/*" // Taken from forge
```

Comment:
Could you put it into forge sub-package? `chunkio/async` could also contain parts of our own code.